include/sapi/tss2_tpm2_types.h

```diff
@@ -1185,7 +1185,7 @@ typedef	TPM2_ST TPMI_ST_COMMAND_TAG;
 
 /* Table 68  Definition of TPMS_EMPTY Structure <INOUT> */
 typedef	struct {
-	BYTE empty[0];	 /* a structure with no member  */
+	BYTE empty[1];	 /* a structure with no member  */
```

Comment:
Shouldn't this be just BYTE empty[]; ?

Reply:
Thing about `BYTE empty[]` is, that it's incomplete and sizeof() is not defined.
Since C99 disallows `[0]`, I do favour `[1]`. I just don't want to get into trouble with `malloc`s on TPMS_EMPTY in generated code.

Reply:
The links from @AndreasFuchsSIT pretty much covers the issue. Not sure we can quantify the complexity this will cause in "generated code" since we don't yet have any though. In any case I think the complexity caused by this one type can be absorbed by code generator so I'd like to leave this as-is if @tstruk is OK with it.

Reply:
Yes, BYTE empty[1]; looks good to me.